xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -581,11 +581,13 @@ void CMusicInfoTag::SetArtist(const CArtist& artist)
 {
   SetArtist(artist.strArtist);
   SetAlbumArtist(artist.strArtist);
+  SetMusicBrainzArtistID({ artist.strMusicBrainzArtistID });
   SetGenre(artist.genre);
-  m_dateAdded = artist.dateAdded;
-  m_iDbId = artist.idArtist;
-  m_type = MediaTypeArtist;
-  m_bLoaded = true;
+  SetMood(StringUtils::Join(artist.moods, " / "));
```

Comment:
We should probably use m_musicItemSeparator from advancedsettings here?
Not really part of this PR just came to mind.